lib/models/user-whitelist.js

```diff
@@ -13,4 +14,28 @@ util.inherits(UserWhitelist, Base);
 
 require('../extend-with-factories')(UserWhitelist);
 
+UserWhitelist.prototype.isInTrial = function () {
+  return moment(this.attrs.trialEnd) > moment().utc();
```

Comment:
is there a reason you did not use [isBefore](http://momentjs.com/docs/#/query/is-before/)